test/pjrt/test_experimental_pjrt_tpu.py

```diff
@@ -34,6 +34,8 @@ def setUp(self):
     # sub-processes will not be able to initialize the client witht the correct
     # settings.
 
+  @absltest.skipIf(tpu.get_tpu_env()['ACCELERATOR_TYPE'] == 'v3-8',
+                   'This test is not currently supported on v3 TPUVMs.')
   def test_xla_devices_multiprocess(self):
     accelerator_devices = {
         'v3-8': {
```

Comment:
do you need to remove 'v3-8' here since the test is not supported on v3?

Reply:
@will-cromar Do you think we will ever support this test on v3? If not, we may as well remove it from `accelerator_devices` here. If we do plan on supporting it eventually, we should probably keep the code.

Reply:
Go ahead and remove the `v3-8` test config -- it's based on a bad assumption of mine. Then change the `NotImplementedError`s to `skip`s instead so we don't fail out when tests are not implemented yet.

Reply:
Done